samples/drivers/led_strip/boards/nucleo_u575zi_q.overlay

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
I agree that commit restructuring is required, but can we do it later to ease up review cycle?
for e.g. i'm planning to push all review point fixes in a separate commit so that reviewer can easily find them, and then once everything is passed i'll restructure commits.
I can do it with each review iteration too, if that is a preferred approach

Reply:
> I agree that commit restructuring is required, but can we do it later to ease up review cycle? for e.g. i'm planning to push all review point fixes in a separate commit so that reviewer can easily find them, and then once everything is passed i'll restructure commits. I can do it with each review iteration too, if that is a preferred approach

Please don't do that. Fix and amend your commits at each iteration. Reviewers will be able to track changes.

Reply:
commits rearranged, please review